nautobot/core/settings.py

```diff
@@ -814,7 +814,7 @@ def UI_RACK_VIEW_TRUNCATE_FUNCTION(device_display_name):
     # "NautobotHyperlinkedRelatedField": {"type": "string", "readOnly": True},
     # type=string results in a free text field; also not what we want. For now,
     # however, this will keep things moving so the unit tests pass.
-    "NautobotHyperlinkedRelatedField": {"type": "string", "format": "uuid"},
+    "NautobotHyperlinkedRelatedField": {"type": "object"},
```

Comment:
My only concern here is what implications this will have for the field-level rendering and validation in RJSF and how it behaves in the React UI. We very likely will have to do some sort of custom template for this field mapping.